services/powershellgallery/powershellgallery.service.js

```diff
@@ -135,4 +140,78 @@ class PowershellGalleryDownloads extends BaseXmlService {
   }
 }
 
-module.exports = { PowershellGalleryVersion, PowershellGalleryDownloads }
+class PowershellGalleryPlatformSupport extends BaseXmlService {
+  static get category() {
+    return 'platform-support'
+  }
+
+  static get route() {
+    return {
+      base: 'powershellgallery/ps',
+      pattern: ':packageName',
+    }
+  }
+
+  static get examples() {
+    return [
+      {
+        title: 'PowerShell Gallery',
+        namedParams: { packageName: 'Az.Storage' },
+        staticExample: this.render({
+          platforms: ['windows', 'macos', 'linux'],
+        }),
+      },
+    ]
+  }
+
+  static render({ platforms }) {
+    return {
+      label: 'platform',
+      message: platforms.join(' | '),
+    }
+  }
+
+  async handle({ packageName }) {
+    const packageData = await fetch(this, {
+      packageName,
+    })
+    const { 'd:Tags': tagStr } = packageData
+
+    const platforms = new Set()
+    const tagArr = tagStr.split(' ')
+
+    for (const tag of tagArr) {
+      switch (tag.toLowerCase()) {
+        // Look for Windows
+        case WINDOWS_TAG_NAME:
+          platforms.add(WINDOWS_TAG_NAME.toLowerCase())
+          break
+
+        // Look for MacOS
+        case MACOS_TAG_NAME:
+          platforms.add(MACOS_TAG_NAME.toLowerCase())
+          break
+
+        // Look for Linux
+        case LINUX_TAG_NAME:
+          platforms.add(LINUX_TAG_NAME.toLowerCase())
+          break
+
+        default:
+          break
+      }
+    }
+
+    if (platforms.size === 0) {
+      platforms.add('unknown')
+    }
```

Comment:
(This comment disappeared, not sure why.) Let's make this `'not specified'` to make it clear the package is valid but the platform is missing from the metadata.

Reply:
fixed